content/Resources/GPU_Resources/CMS_Resources/ml_cern_ch.md

```diff
@@ -0,0 +1,16 @@
+# ml.cern.ch
+
+[ml.cern.ch](https://ml.cern.ch) is a `Kubeflow` based ML solution provided by CERN. 
+
+## `Kubeflow` 
+[`Kubeflow`](https://www.kubeflow.org/docs/started/introduction/) is a Kubernetes based ML toolkits aiming at making deployments of ML workflows simple, portable and scalable. In Kubeflow, *pipeline* is an important concept. Machine Learning workflows are discribed as a Kubeflow *pipeline* for execution.
+
+## How to access
+[`ml.cern.ch`](https://ml.cern.ch) only accepts connection from CERN internet. Therefore, if you are outside of CERN a network tunneling will be needed (e.g. via `ssh -D` dynamical port forwarding as proxy). The main website are shown below.
```

Comment:
"only accepts connections from within the CERN network. Therefore, if you are outside of CERN, you will need to use a network tunnel (eg. via `ssh -D` dynamic port forwarding as proxy)..."

Is it really the `-D` option that you want of is it `-L` as is recommended [here](https://linuxize.com/post/how-to-setup-ssh-tunneling/)?

Reply:
Thanks for the rephrase. Here the `-D` option is used to create a SOCKS5 proxy so one can browse CERN internet through it. With a SOCKS5 proxy, the data will be viewed as within CERN internet.